client/src/components/OauthCallback.jsx

```diff
@@ -0,0 +1,94 @@
+// this component will eventually be rendered in the Dashboard page
+
+import React from "react";
+import GoogleButton from "react-google-button";
```

Comment:
Remove unused files

Are you using eslint?

Reply:
Done